asserts/system_user_test.go

```diff
@@ -97,7 +97,6 @@ func (s *systemUserSuite) TestDecodeOK(c *C) {
 	c.Check(systemUser.SSHKeys(), DeepEquals, []string{"ssh-rsa AAAABcdefg"})
 	c.Check(systemUser.Since().Equal(s.since), Equals, true)
 	c.Check(systemUser.Until().Equal(s.until), Equals, true)
-	c.Check(systemUser.UserExpiration().Equal(s.until), Equals, true)
```

Comment:
this should become the IsZero check, no?

Reply:
Good point!